ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -478,6 +478,36 @@ function check_upgrade_status() {
     fi
 }
 
+# update qe-app-registry catalogsource to correct version after OCP is upgraded
+function update_aosqe_index() {
+    if oc get catalogsource qe-app-registry -n openshift-marketplace >/dev/null 2&1 ; then
+        ocp_channel=$(oc get clusterversion/version -o jsonpath="{.spec.channel}")
+        if [[ $? == 0 ]] ; then
+            ocp_version=${ocp_channel//stable-}
```

Comment:
This channel value is not `stable-` prefix sometimes, so I guess we can get the version by using
```console
oc version | grep "Server Version" |awk '{print $3}'
```

Reply:
Acked